pkgs/applications/networking/cluster/rke2/update-script.sh

```diff
@@ -1,49 +1,41 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl git gnugrep gnused yq-go nix-prefetch
+#!nix-shell -i bash -p curl git gnugrep gnused yq-go nix-prefetch go
 
-set -x -eu -o pipefail
+set -x -e -o pipefail
```

Comment:
Why not keep `-u`? It's useful to detect typos and unused variables.

Reply:
Inside of this script we execute another script from the Rancher repository. There are some unused variables in there, which is ok. With `set -x -eu -o pipefail` we are unable to execute that script successfully.

Reply:
We now explicitly `set +u` before execution of the mentioned script and afterwards revert to `set -u`.